### ISSUE TITLE
What mean 'sourse'? 

### ISSUE THREAD

**Issue by vkhv** [x1024/vim-cyrillic#1]
Куда это нужно ставить?) В .vimrc? В папку plugin?

**Comment by metala** [comment]
Yep, just edit your `.vimrc` or related configuration file and add 
```
source /path/to/vim_cyrillic.vim
```

However, unfortunately for you it may not fit our keyboard layout. As far as I can tell from the file it's the Bulgarian Phonetic keyboard layout.
I would recommend you that you make yourself your own keyboard remapping that suits your specific cyrillic layout.
